templates/api/kubernetes/version.go.tpl

```diff
@@ -10,7 +10,7 @@
 package {{ .version }}
 
 //nolint:lll //Why: Long shell script
-//go:generate /usr/bin/env bash -c "pushd ../../..{{if not (empty .package)}}/..{{end}} >/dev/null 2>&1 && ./scripts/shell-wrapper.sh gobin.sh sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0 object paths=./api/k8s/{{ .package }}/{{ .version }}; popd >/dev/null 2>&1"
+//go:generate /usr/bin/env bash -e -c "pushd ../../..{{if not (empty .package)}}/..{{end}} >/dev/null 2>&1 && ./scripts/shell-wrapper.sh gobin.sh sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0 object paths=./api/k8s/{{ .package }}/{{ .version }}; popd >/dev/null 2>&1"
```

Comment:
I think alternately:
```suggestion
//go:generate /usr/bin/env bash -c "pushd ../../..{{if not (empty .package)}}/..{{end}} >/dev/null 2>&1 && ./scripts/shell-wrapper.sh gobin.sh sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0 object paths=./api/k8s/{{ .package }}/{{ .version }} && popd >/dev/null 2>&1"
```
(the asymmetrical `&&` is weird either way)

Reply:
Yup that works too.  I'll leave it up to y'all to tweak/merge as you see fit.  For now I will just manually fix in my repo.